Cartfile

```diff
@@ -1,4 +1,5 @@
-github "ReactiveX/RxSwift" == 5.0
+github "ReactiveX/RxSwift" == 5.1.1
```

Comment:
When a version of RxSwift is `5.0`, some crash happened.
- Produced RxSwift.framework via Carthage, store in `Carthage/iOS/Build/Static/*`
- Not worked databinding between ui and tableView
Error is as follows.

```
failed to demangle superclass of DelegateProxy from mangled name 'So16_RXDelegateProxyC'
```